docs/references/faq.md

```diff
@@ -135,3 +135,147 @@ out_batch = pa.Table.from_arrays([new_data, batch.column('ColumnToKeepAsIs')], s
 Use the newest version of the Python SDK which contains fixes that concern date and time. Use the `arrow_scalar_date32_to_py` (`ayx_python_sdk.providers.amp_provider.utilities.conversions`) in case you use the tool in an older version of Designer.
 
 Older versions of Designer process the date incorrectly. This function ensures dates are set correctly for those versions of Designer, otherwise the function simply does nothing.
+
+## _"Why does the CLI use Python?"_
+
+Since we were already developing a Python SDK, it made sense to use
+Python for the CLI, too. Python lets us leverage the [doit automation
+tool](https://pydoit.org/) to build the CLI. We chose to use
+[Typer](https://typer.tiangolo.com/) in combination with doit because it lets us map CLI
+commands to doit tasks.
+
+
+## _"Can I use the CLI and SDK behind a firewall?"_
+
+### _"Do I need to whitelist anything to use the CLI?"_
+
+To download and install the CLI, be notified of a newer version of the
+CLI, or check for updates to the CLI, you need to whitelist
+[https://pypi.org/](https://pypi.org/).
+Otherwise, feel free to use the CLI completely offline.
```

Comment:
TW review:

To download and install the CLI, receive notifications of newer versions of the
CLI, or check for updates to the CLI, you need to whitelist
[https://pypi.org/](https://pypi.org/).
Otherwise, feel free to use the CLI completely offline.

Reply:
fixed